src/video/SDL_blit_A.c

```diff
@@ -1298,13 +1201,15 @@ static void BlitNtoNPixelAlpha(SDL_BlitInfo *info)
     SDL_PixelFormat *dstfmt = info->dst_fmt;
     int srcbpp;
     int dstbpp;
+    int freeFormat;
```

Comment:
SDL_bool?

Reply:
Thanks, updating all Booleans I've added to use that.

Reply:
This should be addressed now.